src/iconLists.ts

```diff
@@ -1,9 +1,19 @@
 /* NOTE: Do not modify this file manually! It is generated by `createComponents.js` when running `npm run generate` */ 
 
 export const baseIcons = ["Alert","AlertOff","AlignLeft","ApplicationProfile","Archive","ArrowUp","ArrowRight","ArrowDown","ArrowLeft","ArrowheadDown","ArrowheadUp","Attachment","Authorise","Basket","BasketAdd","Bold","Calendar","CalendarChecked","Chat","ChatHeart","ChatQuestion","Check","CheckCircleFilled","CheckCircle","CheckSelected","ChevronUp","ChevronRight","ChevronDown","ChevronLeft","ChevronCircleUp","ChevronCircleRight","ChevronCircleDown","ChevronCircleLeft","ChevronDoubleRight","ChevronDoubleLeft","Clock","Close","Compare","CompareRemove","ControlPrevious","ControlPlay","ControlNext","Copy","Disabled","Download","Edit","ErrorFilled","Error","ExpandableMinus","ExpandablePlus","FileGeneric","Fullscreen","Grid","HeartFilled","Heart","HelpFilled","Help","Hint","History","Image","InfoFilled","Info","Internet","Isa","Italics","Link","LinkBreadcrumb","LinkExternal","LinkList","ListBulleted","ListNumbered","Login","Logout","MailSend","MapLayers","MapLocationFilled","MapLocation","MapMyLocation","MapRoute","Map","Menu","Merge","Message","Minus","OptionsVertical","Peek","Pin","Phone","Plus","Preview","Print","Quotes","RadioButtonOn","Refresh","Registers","Remove","Reply","Rows","Save","Search","Settings","SignLanguageContent","Split","Star","StarFilled","SubDirectory","SwapRounded","SwapVertical","TransportBicycle","TransportBus","TransportCar","TransportWalk","Upload","Warning","Window"]
+const baseIconsList = ["Alert","AlertOff","AlignLeft","ApplicationProfile","Archive","ArrowUp","ArrowRight","ArrowDown","ArrowLeft","ArrowheadDown","ArrowheadUp","Attachment","Authorise","Basket","BasketAdd","Bold","Calendar","CalendarChecked","Chat","ChatHeart","ChatQuestion","Check","CheckCircleFilled","CheckCircle","CheckSelected","ChevronUp","ChevronRight","ChevronDown","ChevronLeft","ChevronCircleUp","ChevronCircleRight","ChevronCircleDown","ChevronCircleLeft","ChevronDoubleRight","ChevronDoubleLeft","Clock","Close","Compare","CompareRemove","ControlPrevious","ControlPlay","ControlNext","Copy","Disabled","Download","Edit","ErrorFilled","Error","ExpandableMinus","ExpandablePlus","FileGeneric","Fullscreen","Grid","HeartFilled","Heart","HelpFilled","Help","Hint","History","Image","InfoFilled","Info","Internet","Isa","Italics","Link","LinkBreadcrumb","LinkExternal","LinkList","ListBulleted","ListNumbered","Login","Logout","MailSend","MapLayers","MapLocationFilled","MapLocation","MapMyLocation","MapRoute","Map","Menu","Merge","Message","Minus","OptionsVertical","Peek","Pin","Phone","Plus","Preview","Print","Quotes","RadioButtonOn","Refresh","Registers","Remove","Reply","Rows","Save","Search","Settings","SignLanguageContent","Split","Star","StarFilled","SubDirectory","SwapRounded","SwapVertical","TransportBicycle","TransportBus","TransportCar","TransportWalk","Upload","Warning","Window"] as const
```

Comment:
Do all of these really have to be reiterated here for the typecasting?

Reply:
In this case it is actually [const assertion](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions) and not type casting.

If we can agree that `baseIcons` can be repurposed, there is no need to generate same icon list twice.
Currently doing the extra mile to not cause changes in possible use cases.

Reply:
I don't see a reason why a separate (and duplicate) `*IconsList` array is needed there. I think we can go with the `*Icons` array only, and build the keys export from that.